Modules/_threadmodule.c

```diff
@@ -77,6 +91,24 @@ ThreadHandle_dealloc(ThreadHandleObject *self)
     Py_DECREF(tp);
 }
 
+void
+_PyThread_AfterFork(struct _pythread_runtime_state *state)
+{
+    PyThread_ident_t current = PyThread_get_thread_ident_ex();
```

Comment:
Can you add a comment to this GH issue? So that people find the rationale for this easily.